docs/usage.md

```diff
@@ -44,20 +45,74 @@ TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
 
+## Assembler options
+
+By default, the pipeline uses Megahit (i.e. `--assembler  megahit`) to assemble the cleaned and trimmed FastQ reads to create the reference genome. Megahit is fast and it requires a not a lot of memory to run, typically is suggested to be used with prokaryotic samples. The pipeline allows you to choose another assembler RNAspades, (i.e. `--assembler rnaspades` ), that is usually suggested to use for eukaryotic samples.
+
+> N.B. you can use `Megahit` for eukaryotic samples too, we just suggest what is the best option according to our experience (literature?).
+
+## Orf caller options
+
+By default, the pipeline uses prodigal (i.e. `--orf_caller prodigal` ) to generate the genome feature file (.gff) and to generate gene structure from the assembly.
+
+Other orf caller options for running the pipeline are:
+
+- Prokka (`--orf_caller prokka`)
+
+- Transdecoder (`--orf_caller transdecoder`)
+
+> N.B. Prokka and prodigal are suggested to run with prokaryotes while transdecoder is specific for eukaryotes.
+
+## Taxonomical annotation options
+
+Metatdenovo uses `EUKulele` as main program for taxonomy annotation. EUKulele can run by using different reference datasets. The default dataset is PhyloDB (i.e. `--eukulele_db phylodb` ) which is suggested mainly for prokaryotes.
+
+Other databases options for running the pipeline are:
+
+- Mmetsp (i.e. `--eukulele_db mmetsp`)
+
+- GTDB (i.e. `--eukulele_db gtdb`) [not available yet]
+
+PhyloDB and GTDB are recommended for prokaryotic datasets while mmetsp for eukaryotes although PhyoDB can be also recognize eukaryotes and can be used for this purpose.
+
+If you already have these databases ready in your working directory,  you can redirect to the folder so the pipeline will not download the database
+(e.g. `--eukulele_dbpath your/path/database/` 
+> Please, check the EUKulele documentation for more information about the databases cited [HERE](https://eukulele.readthedocs.io/en/latest/#)
+
+## Functional annotation options
+
+By default, metatdenovo will not perform any functional annotation. In order to run it you will need to specify one of the options that we provide.
+
+This options are:
+
+- [Eggnog](https://github.com/eggnogdb/eggnog-mapper/wiki) (i.e. `--eggnog`)
+
+- [Hmmsearch](http://eddylab.org/software/hmmer/Userguide.pdf) (i.e. `--hmmsearch`)
+
+- [Rundbcan3](https://github.com/linnabrown/run_dbcan) (i.e. `--run_dbcan`)
+
+All the options can run in the same time (e.i. `extflow run main.nf  -profile test,docker --eggnog --hmmsearch --rundbcan`) but each program has it’s own option that you will need to read carefully before run the pipeline. You can find the different options in [parameters]() page and read about the programs from their own website.
+
+## Example pipeline command with some features
+
+`nextflow run nf-core/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog --run_dbcan`
```

Comment:
Maybe we should have "lnuc-eemis" instead of "nf-core" for now? Otherwise we should change "lnuc-eemis" to "nf-core" in my previous comment..